mergeable/src/Mergeable.ts

```diff
@@ -0,0 +1,28 @@
+export async function checkMergeability(slack: Slack, gitHub: GitHub, secrets: Secrets): Promise<any> {
+    const result: Result = await fetchGitHubMergeability(gitHub, secrets.gitHub)
+    const recipient: string = secrets.slack.recipient
+    return await notifySlack(slack)(recipient)(result)
+}
+
+async function fetchGitHubMergeability(gitHub: GitHub, secrets: GitHubSecrets): Promise<Result> {
+    const prs = await gitHub.fetchOpenPullRequests(secrets)
+    const unmergeablePrs = prs.map((pr: any) => pr.data).filter((pr: any) => !pr.mergeable)
+    const isCalculating: boolean = prs.filter((pr: any) => pr.data.mergeable === null).length > 0
```

Comment:
https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Array/some instead of `filter`